cafebabel/templates/articles/macros.html

```diff
@@ -17,13 +17,16 @@
 {%- endmacro %}
 
 {%- macro tags_fields(tags, article=None) -%}
-  <div>
+  <div class=tags>
     <label for=tag-1>Tags</label>
-    <input list=tags name=tag-1 {% if article and article.tags and article.tags[0] %}value={{ article.tags[0].name }}{% endif %}>
-    <input list=tags name=tag-2 {% if article and article.tags and article.tags[1] %}value={{ article.tags[1].name }}{% endif %}>
-    <input list=tags name=tag-3 {% if article and article.tags and article.tags[2] %}value={{ article.tags[2].name }}{% endif %}>
-    <input list=tags name=tag-4 {% if article and article.tags and article.tags[3] %}value={{ article.tags[3].name }}{% endif %}>
-    <input list=tags name=tag-5 {% if article and article.tags and article.tags[4] %}value={{ article.tags[4].name }}{% endif %}>
+    <div class=tag-container><input name=tag-new><button class=add></button></div>
+    {% if article and article.tags %}
+    {% set count = 1 %}
+      {% for tag in tags['en'] %}
+        <button class=remove>+</button><input list=tags name=tag-{{ count }} value={{ article.tags.name }}>
```

Comment:
you may need the quotes for the value here at it may contain space or other special character

Reply:
From a code point of view – without the interface – it looks surprising to have a "+" with a class _remove_